pkg/serverless/trace/propagation/extractor_test.go

```diff
@@ -404,6 +523,112 @@ func TestExtractorExtract(t *testing.T) {
 	}
 }
 
+func TestExtractorExtractPayloadJson(t *testing.T) {
```

Comment:
This test uses several of the example event payloads from our test fixtures. Since extracting from the message attributes is pretty open ended, this will ensure we are able to correctly extract from some event payloads gathered from the wild.